src/plugins/es_ui_shared/docs/docs/form_lib/core/about.md

```diff
@@ -0,0 +1,77 @@
+---
+id: about
+title: Core
+sidebar_label: Getting started
+---
+
+The core exposes the main building blocks (hooks and components) needed to build your form.
+
+It is important to note that the core **is not** responsible of the rendering in the UI. Its resposibility is to return form and fields **state and handlers** that you will then have to connect to your React components. With that said, as in Kibana we work with [the EUI component library](https://elastic.github.io/eui), we have created special fields component that wrap the EUI components where the connection with the form lib is already done for you.  
+The takeaway is: the core of the form lib is agnostic of any UI rendering the form.
+
+## Getting started
+
+The three required component to build a form are
+
+- `useForm()` hook to declare a new form
+- `<Form />` component that will wrap your form and create a context for it
+- `<UseField />` component to declare a field
+
+Let's see them in action before going into details
+
+```js
+import { useForm, Form, UseField } from 'src/plugins/es_ui_shared/public';
+
+export const UserForm = () => {
+  const { form } = useForm(); // 1
+
+  return (
+    <Form form={form}> // 2
+      <UseField path="name" /> // 3 
+      <UseField path="lastName" />
+
+      <button onClick={form.submit}>Submit</button>
+    </Form>
+  );
+};
+```
+
+1. We use the `useForm` hook to declare a new form.
+2. We then wrap our form with the `<Form />` component, providing the `form` that we have just created.
+3. Finally, we declared two fields with the `<UseField />` component, providing a unique `path` for each one of them.
+
+If you were to run this code in the browser and click on the "Submit" button nothing would happen as we haven't defined yet any handler to execute when submitting the form. Let's do that now along with providing a `UserFormData` interface to the form, which we will get back in our `onSubmit` handler.
+
+```js
+import { useForm, Form, UseField, FormConfig } from 'src/plugins/es_ui_shared/public';
```

Comment:
The content on this page is terrific! I do think this is another opportunity where we can combine the examples though, to speed up the pace of information transfer a bit.

Reply:
I do prefer in this case to go peacefully and not introduce too many concepts in one example.